docs/source/conf.py

```diff
@@ -99,6 +99,7 @@
 
 
 # -- Options for HTML output ----------------------------------------------
+add_module_names = False
```

Comment:
I'm not completely sure this is what we want globally, but let's do this for now and see if there are any downsides